File: Logic_HandsOn/C++/Problems on Digits,String,Array,Pointers etc/program89.cpp
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Accept Character from user and check whether it is small case or not (a-z).
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////


#include<iostream>
using namespace std;

class Solution
{
	private:
		char ch;
		
	public:
		Solution(char ch='\0')
		{
			this->ch = ch;
		}

		bool ChkSmall()
		{
			if(ch>='a' && ch<='z')
			{
				return true;
			}
			else
			{
				return false;
			}


		}


};

int main()
{
	char ch='\0';
	bool bRet=false;

	cout<<"Enter the character: ";
	cin>>ch;

	Solution *dObj = new Solution(ch);
	bRet = dObj->ChkSmall();

	if(bRet==true)
	{
		cout<<"It is Small\n";
	}
	else
	{
		cout<<"It is not Small\n";
	}

	delete dObj;
	return 0;
}

/*
OUTPUT:
Enter the character: g
It is Small

*/

File: Logic_HandsOn/Java/Problems on Array,Pattern etc/Program89.java
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Accept Character from user and check whether it is small case or not (a-z).
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Solution
{

	public char ch;

	public Solution(char ch)
	{
		this.ch = ch;
	}

	public boolean ChkSmall()
	{	
		
		if(ch>='a' && ch<='z')
		{
			return true;
		}
		else
		{
			return false;
		}
	}


}

class Program
{
	public static void main(String[] args)
	{
		Scanner sObj = new Scanner(System.in);
		boolean bRet = false;

		System.out.println("Enter character:");
		char ch = sObj.next().charAt(0);

		Solution obj = new Solution(ch);
		bRet = obj.ChkSmall();

		if(bRet == true)
		{
			System.out.println("It is Small");
		}
		else
		{
			System.out.println("It is not Small");
		}
	}
}

/*
OUTPUT:

Enter character:
a
It is Small

*/
